### PR TITLE
 docs:modify some spelling mistakes in the doc

### PR DESCRIPTION
**Pull request by yunfeiyangbuaa**
### Ⅰ. Describe what this PR did
modify  the spelling errors in the docs

### Ⅱ. Does this pull request fix one issue?

fixes：#628

### CODE REVIEW

**Comment by allencloud** [comment]
CI fails with the markdown validation:
```
./FAQ.md:9: MD022 Headers should be surrounded by blank lines

A detailed description of the rules is available at https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
Exited with code 123
```

Please check https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines

Thanks. @yunfeiyangbuaa

File: FAQ.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 FAQ contains the frequently asked questions about two aspects:
 
-- First, user-facing functionalities.
-- Second, underlying concept and thoery.
+- First, user-facing functionalities...
+- Second, underlying concept and thoery
```
Comment (lowzj) [inline FAQ.md]: `thoery` -> `theory`

File: FAQ.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 FAQ contains the frequently asked questions about two aspects:
 
-- First, user-facing functionalities.
-- Second, underlying concept and thoery.
+- First, user-facing functionalities...
```
Comment (lowzj) [inline FAQ.md]: why do you append 2 dots here?

**Comment by allencloud** [comment]
LGTM

**Comment by allencloud** [comment]
Thank you very much for your contribution. @yunfeiyangbuaa 
What you did means a lot for the project. Please keep contributing and feel free to ask if you need help from the community.

**Pull request merged**